Fabric.Terminology.API/Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -393,59 +405,72 @@ function Publish-TerminologyDacpac() {
-        throw "No RoleID found for role '$role'"
+        throw "No RoleID found for role '$Role'"
     }
 
     return $roleId
 }
 
 
-function Post-ToMds {
+function Invoke-PostToMds {
     param(
-        [ValidateNotNullOrEmpty()][string] $name,
-        [ValidateNotNullOrEmpty()][string] $path,
-        [ValidateNotNullOrEmpty()][string] $discoveryServiceUrl
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $name,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $path,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $discoveryServiceUrl
     )
 
-
-    #$authToken = Read-Host "Log into Atlas was a DosAdmin, and copy the access token from one of the auth headers to MDS and copy it here. This is a temporary workaround"
     # TODO THIS NEEDS TO CHANGE AFTER THE FABRIC UPDATES!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-    $authToken = "eyJhbGciOiJSUzI1NiIsImtpZCI6IkNDQzZFNzIxNTMxNDYzMDQ4OTlEMjY5NEEyRkMzRTNFOEZGRkUxQjEiLCJ0eXAiOiJKV1QiLCJ4NXQiOiJ6TWJuSVZNVVl3U0puU2FVb3Z3LVBvX180YkUifQ.eyJuYmYiOjE1MzYwODEyMjIsImV4cCI6MTUzNjA4MjQyMiwiaXNzIjoiaHR0cHM6Ly9hdGxhc2RlbW8uaHFjYXRhbHlzdC5sb2NhbC9pZGVudGl0eSIsImF1ZCI6WyJodHRwczovL2F0bGFzZGVtby5ocWNhdGFseXN0LmxvY2FsL2lkZW50aXR5L3Jlc291cmNlcyIsImRvcy1tZXRhZGF0YS1zZXJ2aWNlLWFwaSIsImF1dGhvcml6YXRpb24tYXBpIiwiaWRwc2VhcmNoLWFwaSIsInJlZ2lzdHJhdGlvbi1hcGkiLCJ0ZXJtaW5vbG9neS1hcGkiXSwiY2xpZW50X2lkIjoiYXRsYXMiLCJzdWIiOiJIUUNBVEFMWVNUXFxkZXYuYWRtaW4iLCJhdXRoX3RpbWUiOjE1MzYwNzY2NDUsImlkcCI6IldpbmRvd3MiLCJuYW1lIjoiSFFDQVRBTFlTVFxcZGV2LmFkbWluIiwicm9sZSI6WyJIUUNBVEFMWVNUXFxEb21haW4gVXNlcnMiLCJFdmVyeW9uZSIsIkJVSUxUSU5cXFVzZXJzIiwiTlQgQVVUSE9SSVRZXFxORVRXT1JLIiwiTlQgQVVUSE9SSVRZXFxBdXRoZW50aWNhdGVkIFVzZXJzIiwiTlQgQVVUSE9SSVRZXFxUaGlzIE9yZ2FuaXphdGlvbiIsIkhRQ0FUQUxZU1RcXFdpRmkgVXNlcnMiLCJIUUNBVEFMWVNUXFxFbnRlcnByaXNlQXJjaGl0ZWN0IiwiQlVJTFRJTlxcQWRtaW5pc3RyYXRvcnMiLCJBdXRoZW50aWNhdGlvbiBhdXRob3JpdHkgYXNzZXJ0ZWQgaWRlbnRpdHkiLCJBdGxhc0RlbW9cXFZTVFNfQWdlbnRTZXJ2aWNlX0c4NzQ3NCIsIkF0bGFzRGVtb1xcVlNUU19BZ2VudFNlcnZpY2VfR2M2NTNmIl0sInNjb3BlIjpbIm9wZW5pZCIsInByb2ZpbGUiLCJmYWJyaWMucHJvZmlsZSIsImRvcy9tZXRhZGF0YSIsImZhYnJpYy9hdXRob3JpemF0aW9uLnJlYWQiLCJmYWJyaWMvYXV0aG9yaXphdGlvbi53cml0ZSIsImZhYnJpYy9hdXRob3JpemF0aW9uLmRvcy53cml0ZSIsImZhYnJpYy9hdXRob3JpemF0aW9uLm1hbmFnZWNsaWVudHMiLCJmYWJyaWMvaWRwcm92aWRlci5zZWFyY2h1c2VycyIsImZhYnJpYy9pZGVudGl0eS5yZWFkIiwiZG9zL3ZhbHVlc2V0cyJdLCJhbXIiOlsiZXh0ZXJuYWwiXX0.AeMjjE4fKF6wmoP2LyjKsfTGub1_nUU11h0TsiByqnpw7ZH1319kLL0QLQ-xO6PZuBqekdkAjSY8RAGLHfobaBPcInhlCbwtePU1v0ge-HGwb4q5ezf0HAW-TInL3SB_714o-ZXwdeZGjBchJVm3Xq24-mE2QBGih0H-lob-_Oxxr8wNLtfN0hmstKpwWqdbKnzwQ-9O6D7VOGzGdJgcqcnzcc-GgzcPwz1b6LHk-_v9W90uyXem5o50bDqRZsVYkKwA3kXPr4EUUgPeSa2HzpcN5GvztUNfkLGpRaqDSUbot1IqvP20tZh28flEXpY5nlvfXcOO_IRd46u7sYSMCA"
+    $authToken = Read-Host "Log into Atlas was a DosAdmin, and copy the access token from one of the auth headers to MDS and copy it here. This is a temporary workaround"
```

Comment:
:+1: I think this is a much better solution than a stale token that guaranteed causes errors